test/tests/nek_standalone/conj_ht/tests

```diff
@@ -1,10 +1,23 @@
 [Tests]
-  [test]
+  [nek_example]
+    type = CSVDiff
+    input = nek.i
+    csvdiff = nek_out.csv
+    min_parallel = 2
+    mesh_mode = 'replicated'
```

Comment:
Ah, sorry just noticed this as well - why is the `mesh_mode` here set to replicated? I think this input should also work in distributed mesh cases.

You can run the test suite in distributed mode with

```
./run_tests --parallel 4 --distributed-mesh
```

where 4 is the number of MPI ranks per test.

Reply:
I think I just copied this from another test and didn't change this, thanks for pointing it out.